src/app/util/process-global-message.cpp

```diff
@@ -548,11 +540,11 @@ bool emAfProcessGlobalCommand(EmberAfClusterCommand * cmd)
 #if defined(EMBER_AF_PLUGIN_SIMPLE_METERING_SERVER)
         emberAfPluginSimpleMeteringClusterReadAttributesResponseCallback(clusterId, message + msgIndex, msgLen - msgIndex);
 #endif
-
-        if (!emAfReadAttributesResponse(clusterId, message + msgIndex, msgLen - msgIndex))
-        {
-            emberAfSendDefaultResponse(cmd, EMBER_ZCL_STATUS_SUCCESS);
-        }
+        // TODO Commented out since not implemented and was defaulted to 0 A.K.A always true in this context
```

Comment:
Please make sure there's an issue tracking this.

Reply:
Issue created